ci/docker/codecov/deploy.Dockerfile

```
@@ -32,6 +33,10 @@ RUN mkdir ${BUILD} && cd ${BUILD} && \
       -DLAPACK_LIBRARY="-L${OPENBLAS_PATH}/lib;openblas" \
       -DSCALAPACK_LIBRARY="-L${SCALAPACK_PATH}/lib;scalapack" \
       -DHWLOC_ROOT=${HWLOC_PATH} \
+      -DDLAI_WITH_ELPA=ON \
+      -DELPA_MODULE_SPEC=$ELPA_PATH/lib/pkgconfig/elpa-${ELPA_VERSION}.pc \
```

According to CMake doc, I'm not sure it is exactly the right thing to do.

The current FindELPA takes the variable does `pkg_search_module(PC_ELPA ${ELPA_MODULE_SPEC})`, which according to https://cmake.org/cmake/help/latest/module/FindPkgConfig.html?highlight=findpkgconfig requires a module spec, not a full filepath or a filename.

```suggestion
      -DELPA_MODULE_SPEC=elpa-${ELPA_VERSION}.pc \
```

At the same time, I think it would be necessary to have the `$ELPA_PATH/lib/pkgconfig` be in the search paths of `pkg-config` (e.g. via environment variable).

---

It's not working... let for now keep the path version.